geopy/geocoders/mapquest.py

```diff
@@ -0,0 +1,188 @@
+from geopy.compat import quote, string_compare, urlencode
+from geopy.geocoders.base import DEFAULT_SENTINEL, Geocoder
+from geopy.location import Location
+from geopy.point import Point
+from geopy.util import logger
+
+__all__ = ("MapQuest", )
+
+
+class MapQuest(Geocoder):
+    """Geocoder using the MapQuest API
+    Documentation at:
+        https://developer.mapquest.com/documentation/geocoding-api/
+    """
+    geocode_path = '/geocoding/v1/address'
+    reverse_path = '/geocoding/v1/reverse'
+
+    def __init__(
+        self,
+        api_key,
+        format_string=None,
+        scheme=None,
+        timeout=DEFAULT_SENTINEL,
+        proxies=DEFAULT_SENTINEL,
+        user_agent=None,
+        ssl_context=DEFAULT_SENTINEL,
+        domain='www.mapquestapi.com',
+    ):
+        """
+        :param str api_key: The API key required by Mapbox to perform
+            geocoding requests. API keys are managed through Mapox's account
+            page (https://www.mapbox.com/account/access-tokens).
```

Comment:
Shouldn't that be MapQuest instead of MapBox?

Reply:
This is correct. I will make another commit this evening to change all these references. Apologies about the sight.

Reply:
All the references to MapBox have been changed to MapQuest.